src/tools.cpp

```diff
@@ -1222,6 +1222,9 @@ const char* getReturnMessage(ReturnValue value)
 		case RETURNVALUE_YOUDONTHAVEREQUIREDPROFESSION:
 			return "You don't have the required profession.";
 
+		case RETURNVALUE_CANNOTMOVEITEMISNOTSTOREITEM:
```

Comment:
we also need the "This item cannot be moved there." when trying to move items from store inbox to anywhere that is not a depot box (or house if its a wrapable item)